scripts/src/primazactl/kube/roles/primazaroles.py

```diff
@@ -10,7 +10,7 @@ def get_primaza_namespace_role(user: str, namespace: str) -> client.V1Role:
             namespace=namespace,
             labels={"app.kubernetes.io/component": "coreV1",
                     "app.kubernetes.io/created-by": "primaza",
-                    "app.kubernetes.io/instance": user,
+                    "app.kubernetes.io/instance": user.replace(":", "-"),
```

Comment:
should this parameter be named `role` or `name` or `role_name`?

```suggestion
                    "app.kubernetes.io/instance": role.replace(":", "-"),
```

Reply:
changed it to `name` which is clearer.